production/src/synch/term.rs

```diff
@@ -0,0 +1,58 @@
+use openapi::apis::configuration::Configuration;
+use openapi::models;
+use openapi::apis;
+use openapi::apis::terms_api;
+use openapi::apis::courses_api;
+use log::error;
+
+// #[allow(unused)]
+// pub async fn get_next_term_data(config: &Configuration) 
+// -> Result<models::Term, String> {
+//     // The Winter X term is considered to be part of associated academic year X.
+//     // But the Spring/Fall X term is considered to be part of associated academic year X + 1.
+
+//     let curr_term_name: String = get_term_name_for_current_term(config).await;
+
+//     if curr_term_name.contains("ERROR") {
+//         Err(String::from("Could not get term data"))
+//     } else {
+
+//     }
+
+// }
+
+#[allow(unused)]
```

Comment:
please actually use these functions in a feature, and then remove this (there's no point in merging dead code)

Reply:
Removed.